pkg/didcomm/transport/http/inbound.go

```diff
@@ -111,7 +113,12 @@ func NewInbound(addr string) (*Inbound, error) {
 		return nil, errors.New("http address is mandatory")
 	}
 
-	return &Inbound{server: &http.Server{Addr: addr}}, nil
+	host, port, err := net.SplitHostPort(addr)
+	if err != nil {
+		return nil, fmt.Errorf("unable to create new Inbound: %s", err)
```

Comment:
I don't see this error being validated in test cases.

Reply:
I check for it in TestStartWithUnsplittableServerAddr. (Note that I used to have "panic" in the name, but that was a mistake so I removed that word)

I check for whether the error text contains "too many colons in address", which is the specific error that gets returned by the SplitHostPort function. Should I also be checking for that "unable to create new Inbound" message too?